src/content/docs/products/splashkit/SplashKit Tutorials/Onboarding/02-linux-installation.md

```diff
@@ -0,0 +1,239 @@
+
+### Download Visual Studio Code
+
+ To install Visual Studio Code, go to the [Visual Studio Code website](https://code.visualstudio.com/) and download the `.deb` file.
+
+![Terminal](/splashkit/linux-installation-fig9.png)
+
+![Terminal](/splashkit/linux-installation-fig10.png)
+
+### Install Visual Studio Code
+
+To install Visual Studio Code, move to the directory where the downloaded `.deb` file by doing:
+
+```bash
+cd ~/Downloads
+```
+
+Then run the following command:
+
+```bash
+sudo dpkg -i code_1.*.deb
+```
+
+![Terminal](/splashkit/linux-installation-fig11.png)
+
+## Installing Necessary Extensions for Visual Studio Code
+
+To install the necessary extensions, keep the terminal open and run the following commands:
+
+### C# Extension
+
+[C# for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp):
+
+```bash
+code --install-extension ms-dotnettools.csharp
+```
+
+![Terminal](/splashkit/linux-installation-fig12.png)
+
+### C# Dev Kit
+
+[C# Dev Kit](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csdevkit):
+
+```bash
+code --install-extension ms-dotnettools.csdevkit
+```
+
+![Terminal](/splashkit/linux-installation-fig13.png)
+
+### Intellicode for C# Dev Kit
+
+[Intellicode for C# Dev Kit](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.vscodeintellicode-csharp):
+
+```bash
+code --install-extension ms-dotnettools.vscodeintellicode-csharp
+```
+
+![Terminal](/splashkit/linux-installation-fig14.png)
+
+### C++ Extension Pack
+
+[C++ Extension Pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools-extension-pack):
+
+```bash
+code --install-extension ms-vscode.cpptools-extension-pack
+```
+
+![Terminal](/splashkit/linux-installation-fig15.png)
+
+### Installing .net SDK
+
+To install the .net SDK, head to the following site and select for your operating system.
+
+[.NET SDK](https://dotnet.microsoft.com/en-us/download?initial-os=linux)
+
+![Terminal](/splashkit/linux-installation-fig16.png)
+
+![Terminal](/splashkit/linux-installation-fig17.png)
+
+Then to install the package, first head to your downloads folder again, then run the following command:
+
+```bash
+sudo apt-get update && \
+  sudo apt-get install -y dotnet-sdk-8.0
+```
+
+![Terminal](/splashkit/linux-installation-fig18.png)
+
+## RESTART Your Computer
+
+After installing all the necessary software, restart your computer to ensure all the changes are applied. Then you're ready to use SplashKit on your Linux machine.
+
+## Beneficial Additional Packages
+
+Some beneficial additional packages to install for working on SplashKit are:
+
+#### Docker
+
+```bash
+sudo apt install docker.io
+sudo usermod -aG docker $USER
+```
+
+#### Python
+
+```bash
+sudo apt install python3 python3-pip
+```
+
+#### node.js and NPM
+
+```bash
+sudo apt install nodejs npm
+```
+
+## Using SplashKit
```

Comment:
From here onward the content is out of scope. This is no longer Linux Installation specific and should belong elsewhere.

Reply:
I think they're beneficial to have as they're most likely going to be needed for working in the splashkit team.

Reply:
What I'm trying to say is that this is not in the windows installation. I agree that is needed to work in the SplashKit team, but it is not in scope with THIS guide. It should be in a standalone guide as part of onboarding.

Reply:
Well this is something that could be organised when the onboarding section is created and added to the site before T3. For now, for the current semester it works for the people who need it to get working on the tasks.

Reply:
Agree that its needed for this trimester. But then why does it not exist in the Windows side? I would argue that only a minority will be using Linux as their OS of choice, with most being Windows or MacOS. Recommend adding it to Windows as well if that is your intent.

This then creates another problem in itself - we now have two identical pieces of information that need to be updated on separate pages, rather than having a single source of truth, which is not in the spirit of what this site is all about.